internal/aez/heater_regulators.go

```diff
@@ -5,35 +5,103 @@ import (
 	"io"
 )
 
+type htr uint16
+
+func (data htr) voltage() float64 {
+	return voltageConstant * float64(data)
+}
+
+func (data htr) resistance() float64 {
+	return 3.3*3900/data.voltage() - 3900
```

Comment:
Elsewhere you have spaces around the /.

Reply:
this is how it should be, formatting is automatic, it might be because of mix of multiplication and subtraction that spacing indicate grouping.